src/pages/WalletsItemModeEnable/WalletsItemModeEnableView.js

```diff
@@ -0,0 +1,48 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+import { t } from 'ttag'
+
+import { walletsPlugin } from 'store/plugins'
+
+import {
+  UserActionInfo,
+  ButtonWithConfirm,
+} from 'components'
+
+import walletsItemModeEnableStyle from './walletsItemModeEnable.m.scss'
+
+export type Props = {|
+  +goBackToWallets: () => any,
+  +walletId: string,
+|}
+
+export class WalletsItemModeEnableView extends PureComponent<Props> {
```

Comment:
It's looks very similar to `WalletsItemModeDisable`, maybe make sense to extract some common things to component?

Reply:
For example?

Reply:
I think that all their common things I reused via `UserActionInfo` component.